app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) == 3 and parts[0] == "mv":
+        mv, file_name, new_file_name = parts
+        if "/" in new_file_name:
+            way, name_file = os.path.split(new_file_name)
+            os.makedirs(way)
```

Comment:
```suggestion
            os.makedirs(way, exist_ok=True)
```

Reply:
```
# os.makedirs() method will raise
# an OSError if the directory
# to be created already exists
# But It can be suppressed by
# setting the value of a parameter
# exist_ok as True
```